aws-ssm-patchbaseline/aws-ssm-patchbaseline.json

```diff
@@ -1,6 +1,6 @@
 {
     "typeName": "AWS::SSM::PatchBaseline",
-    "description": "Resource Type definition for AWS::SSM::PatchBaseline",
+    "description": "The patchbasline creation for aws different types of OS Pipelines",
```

Comment:
the description is not accurate, please check https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-ssm-patchbaseline.html and properly word it.

Reply:
Modified the description of patch-batchbaseline as mention above